recipes/amplisim/build.sh

```diff
@@ -7,6 +7,7 @@ cd htslib
 git reset --hard 7f5136f
 cd htscodecs
 git reset --hard 11b5007
+sed -i "1022c     return getauxval(AT_HWCAP) != 0;" htscodecs/rANS_static4x16pr.c
```

Comment:
IMO this change should be done with a `build > patch`.
Using `sed` would not fail the build if the code is changed later and line 1022 is something else.
Also I guess this change should be done only for Linux-aarch64. It should not be done for Linux-x86_64